validation-test/BuildSystem/extractsymbols-darwin-symroot-path-filters.test

```diff
@@ -0,0 +1,98 @@
+# REQUIRES: standalone_build
+# REQUIRES: OS=macosx
+
+# RUN: %empty-directory(%t)
+# RUN: mkdir -p %t
+# RUN: split-file %s %t
+
+# RUN: mkdir -p %t/destdir/bin
+# RUN: mkdir -p %t/destdir/lib
```

Comment:
Can you also add the comment about the dry-run issue (as I mentioned on the other test) just to provide context for future readers.

Reply:
Added comments to file generations and steps in 1e5f790d06ea4d711daa73dbd9ee48d581be2163